app/konnect/runtime-manager/index.md

```diff
@@ -47,6 +45,92 @@ choose to manually configure runtime instances on the following:
+custom runtime groups. Use multiple groups in one {{site.konnect_short_name}} organization to
+manage runtime instances and their configuration in any groupings you want.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+![runtime groups](/assets/images/docs/konnect/konnect-runtime-groups-example.png)
+> _**Figure 1:** Example runtime group configuration with three runtime groups: the default group, a development group, and a production group. {{site.konnect_short_name}} is the SaaS-managed global management plane that controls all of the groups, while the runtime groups contain self-managed runtime instances._
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions. For more information, see [Administer teams](/konnect/org-management/teams-and-roles).
+
+[Set up and manage runtime groups &rarr;](/konnect/runtime-manager/runtime-groups/manage)
+
+### Runtime group dashboard
+
+For each runtime group, you can view traffic, error rate, and {{site.base_gateway}} service analytics for instances in a runtime group. This allows you to see how much of a runtime group is used. You can also select the time frame of analytics that you want to display.
+
+The following describes the content of the table found on the [Runtime Manager overview](https://cloud.konghq.com/runtime-manager/).
+
+SCREENSHOT HERE
+
+Column | Description
+-------|-------------
+Name | The name of the runtime group. This group name must be unique in the organization.
+ID | A unique ID for the group. This ID is automatically generated when a group is created.
+Type | Types of runtimes that this group contains. Currently, the only supported runtime type is {{site.base_gateway}}.
+
+<!--
+Labels | A list of labels on the runtime group in the form of key:value pairs. -->
+
+[Runtime group dashboard &rarr;](/konnect/runtime-manager/runtime-groups/dashboard)
+
+### Runtime group configuration
+
+For each runtime group, you can spin up runtime instances and configure
+the following {{site.base_gateway}} objects:
+* Gateway services
+* Routes
+* Consumers
+* Consumer Groups
+* Plugins
+* Upstreams
+* Certificates
+* SNIs
+* Vaults
+
+When there are multiple runtime groups, any object configuration only
+applies to the runtime group that it was created in. Consumers and
+their authentication mechanisms don't carry over to other runtime groups.
+
+[{{site.base_gateway}} configuration in {{site.konnect_short_name}} &rarr;](/konnect/runtime-manager/gateway-config)
+
```

Comment:
Can you add another section here?

```## Deleting a runtime group

{:.warning}
> **Warning:** Deleting a group is irreversible. Make sure that you are
certain that you want to delete the group, and that all entities and runtime
instances in the have been accounted for.
To delete a runtime group, you can use the Runtime Manager or the 
{{site.konnect_short_name}} 
[Runtime Groups API](https://developer.konghq.com/spec/cd849478-4628-4bc2-abcd-5d8a83d3b5f2/24c1f98b-ea51-4277-9178-ca28a6aa85d9/).

When a runtime group is deleted, all associated entities are also deleted.
This includes all entities configured in the Runtime Manager for this group.
As a best practice, [back up](/konnect/runtime-manager/backup-restore/) a runtime 
group's configuration before deleting it to avoid losing necessary configuration.

Runtime instances that are still active when the group is deleted will not be
terminated, but they will be orphaned. They will continue processing traffic
using the last configuration they received until they are either connected to
a new runtime group or manually shut down.

You cannot delete the default runtime group.
```
This is coming from my [PR here](https://github.com/Kong/docs.konghq.com/pull/5285/files#diff-7e8d4341f031f78c35b826b476accef254d01bb17caecb58b297320a575c18f9R333). After looking at your topic and comparing, I think this info belongs here, and not buried in that other topic.